### ISSUE TITLE
Empty Data Set description text extending beyond the tableview 

### ISSUE THREAD

**Issue by theblixguy** [waizeiaifu/EmptyDataSet-Swift#7]
# Steps to reproduce

1. Create `UITableView` and add the delegate methods, data source, etc
2. Set the empty data set title, description, etc
3. Load the empty data set
4. Change the description text to something else
5. Reload the empty data set

# What should happen

The description should change to the new text and fit the table view

# What happens instead

The description text extends beyond the table view

![img_0430](https://user-images.githubusercontent.com/1307667/36291611-f2ec626c-12c3-11e8-91d9-134282890f08.png)

# Code

```
func description(forEmptyDataSet scrollView: UIScrollView) -> NSAttributedString? {
        if (noSearchDone && !searchFound) {
            let myAttribute = [NSAttributedStringKey.font: UIFont.systemFont(ofSize: 18.0), NSAttributedStringKey.foregroundColor: UIColor.lightGray]
            let myString = NSMutableAttributedString(string: "Tap the ", attributes: myAttribute)
            let myStringTwo = NSMutableAttributedString(string: " icon (or press the button below) to search for BROOKESBus timings", attributes: myAttribute)
            let imageAttachment = NSTextAttachment()
            imageAttachment.image = UIImage(named: "SearchIcon")?.maskWithColor(color: UIColor(red: 0.0, green: 0.478431, blue: 1.0, alpha: 1))
            imageAttachment.bounds = CGRect(x: 0, y: 0, width: myString.size().height, height: myString.size().height)
            let imageString = NSAttributedString(attachment: imageAttachment)
            myString.append(imageString)
            myString.append(myStringTwo)
            return myString
        } else if (!noSearchDone && !searchFound) {
            let myString = "No information is available for BROOKESBuses going from \"\(departure)\" to \"\(destination)\" at the moment"
            let myAttribute = [NSAttributedStringKey.font: UIFont.systemFont(ofSize: 18.0), NSAttributedStringKey.foregroundColor: UIColor.lightGray]
            let myAttrString = NSAttributedString(string: myString, attributes: myAttribute)
            return myAttrString
        } else {
            return nil
        }
    }
```

**Comment by waizeiaifu** [comment]
I am sorry that I was in the holiday. So I didn't notice your issue. I'll take care of it.

**Comment by waizeiaifu** [comment]
I have fixed it in CocoaPods ~> 4.0.3
